base/replutil.jl

```diff
@@ -151,6 +151,21 @@ function show(io::IO, ::MIME"text/plain", s::String)
     end
 end
 
+function show(io::IO, ::MIME"text/plain", opt::JLOptions)
```

Comment:
Not identical, but they look extremely similar to me. I don't think we really need both.